client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/events.ts

```diff
@@ -71,3 +73,26 @@ export const PATTERN_ASSEMBLER_EVENTS = {
 	LARGE_PREVIEW_ADD_HEADER_BUTTON_CLICK:
 		'calypso_signup_pattern_assembler_large_preview_add_header_button_click',
 } as const;
+
+/**
+ * Tracks prop
+ *  name: assembler_source
+ *  values:
+ *		• virtual-theme
+ * 		• standard-theme
+ *		• blank-canvas-theme
+ */
+export const getAssemblerSource = ( design?: Design ) => {
```

Comment:
WDYT about declaring `getAssemblerSource` in `client/landing/stepper/declarative-flow/internals/analytics/record-design.ts`? That way, the generic file `/client/landing/stepper/declarative-flow/internals/index.tsx` wouldn't have to import from a step-specific file such as `/internals/steps-repository/pattern-assembler/events`?

Reply:
✅ Sounds good, thanks for suggesting it.